lib/navigation.php

```diff
@@ -38,7 +38,7 @@ function gutenberg_register_navigation_post_type() {
 		'public'                => false,
 		'has_archive'           => false,
 		// We should disable UI for non-FSE themes.
```

Comment:
another FSE comment :)
```suggestion
		// We should disable UI for non-block themes.
```